src/test/java/edu/indiana/dlib/amppd/service/BatchServiceTests.java

```diff
@@ -190,18 +191,18 @@ public void shouldBeValid() throws Exception {
         
         BatchValidationResponse response = manifestService.validate("Test Unit", "Test File", users.get(), content);
 
-        if(response.getErrors()!=null) {
-            for(String s : response.getErrors()) {
+        if(response.getValidationErrors()!=null) {
+            for(String s : response.getValidationErrors()) {
                 System.out.println("Should be valid fails validation: " + s);
             }
         }
         
         Assert.assertFalse(response.hasErrors());
         
-        boolean success = batchService.processBatch(response, ampUsername);
+        List<String> errors = batchService.processBatch(response, ampUsername);
         
         
-        Assert.assertTrue(success);
+        Assert.assertTrue(errors.size()==0);
         
 	}
 	
```

Comment:
By the way, not a comment for this PR, but a generic one, you might be aware that the unit tests run on Bamboo use a different DB (amppd-bamboo) that the amppd app on potassium. so if you persist any data in unit tests, it will stay there, unless you delete it after the test. I've run into issues where existing (or non-existing) data cause issues for other or later tests. That's why i added lots of setup and cleanup here and there.

Reply:
I tried to add before and after methods to user tests as well as batch tests to avoid issues with existing data.